.lycheeexclude

```diff
@@ -0,0 +1,166 @@
+https://localhost
+http://127.0.0.1/
+https://127.0.0.1/
+http://127.0.0.1:10002/bar 
+http://127.0.0.1:10002/
+http://opensearch
+https://opensearch
+https://opensearch-dashboards
+http://opensearch-dashboards
+https://opensearch.internal.org/
+https://maps.search-services.aws.a2z.com/
+https://tiles.maps.search-services.aws.a2z.com/ 
+
+# Dummy urls in tests
+http://domain.
+http://www.domain.
+http://somehost
+https://somehost
+http://some.host
+https://some-host.com/
+https://other.some-host.com/
+http://test:user@somehost/
+https://some.another.host/
+http://noone.nowhere.none/
+http://bar
+http://foo
+http://test.com/
+https://files.foobar/
+https://tiles.foobar/
+https://1.1.1.1:9200/
+http://192.168.1.1:1234/
+http://9.8.7.6/
+http://1.2.3.4/
+http://8.8.8.8/
+https://path.to/
+https://example.com/
+http://example.com/
+https://example.org/
+http://some-url/
+http://buildurl/
+https://dryrun/
+https://url/
+http://url/
+http://notfound.svg/ 
+https://validurl/
+https://myopensearch-dashboardsdomain.com
+http://myopensearch-dashboardsdomain.com
+https://other-opensearch-dashboards.external:8080/
+http://myotherdomain.com:5601/
+http://myopensearch-dashboardsdomain.com:5601/
+https://your-cdn-host.com/
+http://not-your-opensearch-dashboards.com/
+http://www.mysite.com/
+http://myserver.mydomain.com:5601/
+https://elastic:changeme@myCloudInstance:9200
+https://myexternaldep.com/
+http://notlocalhost/
+http://site.com/
+http://node-b/
+http://node-a:9200/
+https://node-c/
+https://elsewhere
+https://opensearch:changeme@example.com:9200
+http://elastic:changeme@localhost:61141/
+http://test:user@somehost/
+https://mycloudinstance:9200/
+https://dev-url.co/
+https://extenal.org/_search
+http://plugins.example.com/
+https://build-url/
+http://test.com/
+https://path.to/
+http://site.com/
+http://not-your-opensearch-dashboards.com/
+http://evil.com/
+https://www.opensearch.org/cool/path
+https://www.opensearch.org/redirect
+http://www.opensearch.org/painlessDocs
+https://www.opensearch.org/subscriptions
+https://www.opensearch.org/opensearch-maps-service
+https://www.hostedgraphite.com/
+
+# External urls
+https://www.zeek.org/
+http://google.com/
+https://api.worldbank.org/
+https://vega.github.io/
+http://twitter.com/
+https://twitter.com/
+https://storage.googleapis.com/
+http://tools.ietf.org/
+https://github.com/
+http://github.com/
+http://jsperf
+https://jsperf
+https://gist.githubusercontent.com/
+https://nodejs.org/
+https://www.npmjs.com/
+https://microsoft.github.io/
+http://api.worldbank.org/
+https://f1542b814f674090afd914960583265f.apm.us-central1.gcp.cloud.es.io/
+http://www.matthewcopeland.me/
+http://threedubmedia.googlecode.com/*
+https://developer.mozilla.org/
+https://a.tile.openstreetmap.org/
+http://www.creedthoughts.gov
+https://media-for-the-masses.theacademyofperformingartsandscience.org/
+
+# Exluce files
+file:///*
+git://*
+
+# TODO: Repalce below broken links when prod replacements become available on doc website.
```

Comment:
`Repalce`

Reply:
will fix

Reply:
I wonder if we should revert these back to the elastic links so that they still actually work and devs can reference them as needed until we have a replacement?

Reply:
Yes. That is the plan. I didn't change these elastic links but just added the them to `TODO` list in `.lycheeexclude`. By doing so, lychee won't fail on these broken links but also we have a reference list in .lycheeexclude for future work.